### ISSUE TITLE
PhantomJS 1.9.7 doesn't show icon font

### ISSUE THREAD

**Issue by archer96** [ariya/phantomjs#12132]
Hello,
I'm using PhantomJS to automatically create screenshots of my mobile web app.
I'm also using FontAwesome (an icon font) and everything worked finde until I updated PhantomJS from `1.9.1` to `1.9.7` using Homebrew.

With the latest PhantomJS version I get this for every single icon:

![phantom](https://cloud.githubusercontent.com/assets/1667306/2645739/0871467e-bf36-11e3-927d-72870dba6cb7.png)

OS: MacOS X 10.9.2
node: v0.10.26
PhantomJS: 1.9.7

Any ideas?

Regards,
Andre

**Comment by rjbernaldo** [comment]
Was experiencing the issue with Ionicons, followed @stevematyas suggestion and installed the ttf using Font Book, and everything is now working perfectly. Thank you very much.

**Comment by eugene1g** [comment]
I had a similar-looking issue with V2, but in that scenario it turned out to be a local fault. Posting here in case someones had the same oversight -

We used the standard CSS definition Font Awesome, but removed all physical font files except for .woff and .eot Phantom read the CSS definition and picked .ttf font-file to load (which we found out by looking through access logs). The .ttf file did not exist, so nothing was pulled in - after we restored the ttf file, V2was able to render PDFs with all icons present.

**Comment by GuyMograbi** [comment]
+1 - still happens in 1.9.17..

if anyone is looking for 1.9.1 - we found it at: http://code.google.com/p/phantomjs/downloads/list?can=1&q=&colspec=Filename+Summary+Uploaded+ReleaseDate+Size+DownloadCount

**Comment by abenrob** [comment]
Any leads on this that don't involve installing the fonts? I'm using 1.9.7 on the server for a public webapp. The fonts are located in the public directory for the app, but phantom is somehow missing them.

**Comment by rafaelverger** [comment]
Downgrading to 1.9.1 also worked for me :)

**Comment by w33ble** [comment]
@stevematyas thanks a ton for posting that, I hadn't even thought about overriding the `@font-face` for font-awesome. TrueType fonts still would not work for me, despite my best efforts, but using the SVG font did the trick. 

In my case, I use `execute` to inject a little javascript that loads an external css file to do my overrides. Simply injecting the following css, either as an inject style tag or link to a css file (both worked) was all it took:

```css
@font-face {
  font-family: 'FontAwesome';
  src: url('/path/to/font-awesome.svg') format('svg');
}
```

**Comment by alliejanoch** [comment]
I was having a lot of similar problems, and I just upgraded to the latest version of Phantom (2.1.3) and all my problems have been fixed.

**Comment by Vitallium** [comment]
This was fixed in 1.9.8 version (.ico files only were supported).
